po/glossary/fi.po

```diff
@@ -579,19 +576,19 @@ msgstr "maksu"
 
 #. "An account where no transactions may be posted to; transactions can only be posted to subaccounts of this account, so this account serves as a placeholder in the hierarchy"
 msgid "placeholder"
-msgstr "koontitili"
+msgstr "paikanpitäjä"
```

Comment:
Why? AFAIK it's not an account that accumulates all the transactions in the accounts below, but rather one that just makes up the hierarchy as per the comment. I admit paikanpitäjä isn't ideal either, though. Maybe ”luokkatili” or similar?

Reply:
It collects the accounts even when sums are not there.
I have tried to find time to move my accounting to gnucash, but have not yet found the time. Since I've not used it much, I could get something wrong so it is good you bring up your opinion. I believe some reports go the account tree down only to given depth an do show the sums also for all the accouns under placeholder.
@harski translated this earlier. Maybe we can get a third opinion.

Reply:
Well, kinda, yeah. Reports do of course in some cases show up sums of the subtrees, but I wouldn't call it a feature of the parent account. Also the English term + the comment above it IMO would match much more closely to my suggestion (which, I'll admit, is not too pretty).

I actually wonder if this is something that should also be changed in the English version – IMO a better English word would also be hierarchy account or similar, I remember when I was beginning my use of GnuCash some ten years ago and I think I had some trouble with understanding what this term meant.

If this is the only blocker, then maybe I'll just revert it and we'll get back to it. This isn't even an UI translation but one for the glossary, so it won't change anything anyway.

Reply:
>  I believe some reports go the account tree down only to given depth an do show the sums also for all the accouns under placeholder.

No, there is no limit in the data structure, but some reports can filter details by level.

Reply:
> I actually wonder if this is something that should also be changed in the English version – IMO a better English word would also be hierarchy account or similar, I remember when I was beginning my use of GnuCash some ten years ago and I think I had some trouble with understanding what this term meant.

As you are using weblate, you can file suggestions and comments there against the english text. As we are pretty new there, We still have to work on the process, to improve the source text.

Reply:
I haven't been using Weblate so far, I created the translation using Poedit. But thanks for the tip, I'll maybe suggest this there.

Reply:
This is a tough one. I agree with @ZeiP that the original English term `placeholder` here might not be that accurate in the first place and should be considered to be changed. But for this change set it is what it is, and the discussion for the English term should be had elsewhere.

While I also agree with the principle that the translation shouldn't diverge much from the original text, I don't think the proposed translation `paikanpitäjä`, is a good one. It doesn't sound natural in Finnish *and* it doesn't describe the option well.

> AFAIK it's not an account that accumulates all the transactions in the accounts below, but rather one that just makes up the hierarchy as per the comment.

I understand `koontitili` like @kohtala, something that groups the sub-accounts together. I'm not against changing it, though, if you come up with a better term @ZeiP. What is your reasoning for `luokkatili`?

If we don't come up with a better translation, I vote for leaving this as is.

Reply:
> I agree with @ZeiP that the original English term `placeholder` here might not be that accurate in the first place and should be considered to be changed.

I am inclined to change this attribute to "write protected" or "read-only" after the string freeze is lifted again. That is the real meaning. The original coder tried to be user friendly and saw the use case "group similar accounts in one category". Later users discovered it can be used to write-protect closed accounts, too.

Reply:
Thanks for the insight.